scripts/build-release.ps1

```
@@ -0,0 +1,108 @@
+param(
+    [string]$Target = "",
+    [string]$Profile = "release",
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'Profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)